tests/test_launch_methods/test_jsrun/test_1/cmds

```diff
@@ -0,0 +1,3 @@
+# Task needs to be assigned node 1, physical core 1, HW thread 1
+export OMP_NUM_THREADS=1
+jsrun -a 1 -U resource-file /ccs/proj/csc190/Hello_jsrun/jsrun_layout
```

Comment:
I suggest the following:
```suggestion
jsrun -a 1 -U resource-file /ccs/proj/csc190/Hello_jsrun/jsrun_layout | sort
```

This way all outputs will be sorted and easier to compare with the file you produce. Especially with MPI execution, where the print order is not guaranteed.